branch-diff.js

```diff
@@ -111,6 +111,10 @@ function printCommits (list, format) {
   } else {
     list = list.map((commit) => commitToOutput(commit, format === 'simple', ghId))
   }
+  
+  if (reverse) {
+    list = list.reverse();
+  }
```

Comment:
I actually think `sha` should probably be reversed for default, but I'd like to defer to @rvagg.